ASTree.cpp

```diff
@@ -2306,6 +2389,17 @@ PycRef<ASTNode> BuildFromCode(PycRef<PycCode> code, PycModule* mod)
     return new ASTNodeList(defblock->nodes());
 }
 
+static void append_to_chain_store(PycRef<ASTNode> chainStore, PycRef<ASTNode> item, FastStack& stack, PycRef<ASTBlock> curblock)
+{
+    stack.pop();    // ignore identical source object.
+    chainStore.cast<ASTChainStore>()->append(item);
+    if (stack.top().type() == PycObject::TYPE_NULL) {
```

Comment:
Should be `ASTNode::NODE_INVALID`.

Reply:
I'm not sure how to get that to work - TYPE_NULL is 48 (chr('0')) which is what stack.top() is. E.g. in my debugger
![image](https://user-images.githubusercontent.com/8370997/97096862-eade4f80-16bd-11eb-8196-31d45be12d3e.png)
How do I read that as NODE_INVALID instead?

Reply:
Whoops, you're right.  Sorry for the confusion

Reply:
no worries